README.md

```diff
@@ -1,3 +1,5 @@
+<!-- markdownlint-disable MD024 MD034 MD033 -->
```

Comment:
Any chance you can either also take care of these markdownlint errors (including MD013), or otherwise, please remove the line here.